2-mandatory/4-people-i-know.js

```diff
@@ -402,6 +407,11 @@ This time, I only want the full names of the people who can multitask
 */
 
 let colleaguesWhoCanMultitask = [];
+  friends.forEach((friend) => friend.colleagues.forEach((colleague) => {
```

Comment:
Instead of lots of `forEach` function calls here, how could you use the `Array` functions from the previous excerise to complete this?

Reply:
I can filter array of friends:      let colleaguesWhoCanMultitask = friends
  .filter((friend) => friend.colleagues.some(colleague => colleague.skills.includes("Multi-tasking")));    but after, since there are lot of colleagues, I didn't know how to choose a multitask colleague without if-statement. And the code looks redundant again.